### ISSUE TITLE
Places with multiple phone numbers

### ISSUE THREAD

**Issue by dhekabai** [harsh4870/Justdail-scrapper#2]
Is it possible to open the listing page for each place and scrape multiple numbers if any? It would be a huge help.

**Comment by harsh4870** [comment]
Yes i think you can scrape the multiple numbers and open listing page for each places also.
